sopel/modules/uptime.py

```diff
@@ -10,17 +10,19 @@
 
 import datetime
 
-from sopel.module import commands
+from sopel import plugin
 
 
 def setup(bot):
     if "start_time" not in bot.memory:
         bot.memory["start_time"] = datetime.datetime.utcnow()
 
 
-@commands('uptime')
+@plugin.command('uptime')
+@plugin.example('.uptime', user_help=True)
+@plugin.output_prefix('[uptime] ')
 def uptime(bot, trigger):
-    """.uptime - Returns the uptime of Sopel."""
+    """Return the uptime of Sopel."""
```

Comment:
Hmm, if you search in `sopel/modules` I think you'll find a few other occurrences of `""".` that should get this treatment.

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAH

:fr: :cursing_face:

Reply:
Done.

Reply:
So, uh, now that you did the others, it leads to another consistency check: Should command callable docstrings use imperative mood (this form), or indicative (the `-s` form used in the others you tweaked)?

Obviously Python guidelines say they should be written in the imperative, but I don't really care which. What's important is that our core plugins present themselves consistently in help output.